### PR TITLE
Upgrade nikic/php-parser to 4.15.1

### LINKED ISSUE #1748: Support readonly class being introduced by 8.2
I am already trying PHP 8.2 and I get the following error:

```
Syntax error, unexpected T_READONLY, expecting T_CLASS on line ***
```

The files are like
```php
<?php

namespace MyApp;

final readonly class MyClass
{
    // ...
}
```

Opening this issue since I could not find any related to this problem.

### PR DESCRIPTION
**Pull request by fefas**
Fix #1748

### CODE REVIEW

**Review by sanmai — APPROVED** [review]

File: .github/workflows/mt.yaml [hunk 1]
```diff
@@ -11,7 +11,7 @@ on:
       - master
 
 env:
-  MIN_MSI: 71.35
+  MIN_MSI: 70.41
```
Comment (sanmai) [inline .github/workflows/mt.yaml:14]:
```suggestion
  MIN_MSI: 70.36
```

File: .github/workflows/mt.yaml [hunk 2]
```diff
@@ -11,7 +11,7 @@ on:
       - master
 
 env:
-  MIN_MSI: 71.35
+  MIN_MSI: 70.36
   MIN_COVERED_MSI: 86.50
```
Comment (sanmai) [inline .github/workflows/mt.yaml:15]:
```suggestion
  MIN_COVERED_MSI: 86.48
```

**Review by maks-rafalko — APPROVED** [review]
Thank you @fefas and @sanmai

**Comment by fefas** [comment]
Thx @maks-rafalko and @sanmai for the support.

Just tested and it worked nicely.

**Pull request merged**